k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -420,15 +395,18 @@ function deployConsul()
         --set client.resources.limits.memory="$(extractBlock 'solution.common.resource_allocation.consul.client.resources.limits.memory')" \
         --set client.resources.limits.cpu="$(extractBlock 'solution.common.resource_allocation.consul.client.resources.limits.cpu')" \
         --set client.containerSecurityContext.client.allowPrivilegeEscalation=false \
+        --namespace "${namespace}" \
         --wait
 
     # Patch generated ServiceAccounts to prevent automounting ServiceAccount tokens
-    kubectl patch serviceaccount/consul-client -p '{"automountServiceAccountToken":false}'
-    kubectl patch serviceaccount/consul-server -p '{"automountServiceAccountToken":false}'
+    kubectl patch serviceaccount/consul-client -p '{"automountServiceAccountToken":false}' \
+                                               -n "${namespace}"
```

Comment:
```suggestion
                                               --namespace "${namespace}"
```

There's not a consistent use of `-n` and `--namespace` in this script already. My preference is to always use long options in scripts since they are self documenting.